services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/user_services_preferences/_manager.py

```diff
@@ -21,16 +21,17 @@ class UserServicesPreferencesManager:
     product_name: ProductName
     _preferences_already_saved: bool = False
 
-    async def load_preferences(self) -> None:
+    async def load_preferences(self, application_name: str) -> None:
```

Comment:
Can you please add this as an attribute to the UserServicesPreferencesManager dataclass?
This way we can remove it form here